### ISSUE TITLE
Gradle 4.8-rc-1 compatibility issue.

### ISSUE THREAD

**Issue by artem-zinnatullin** [bugsnag/bugsnag-android-gradle-plugin#109]
Gradle recently [published 4.8-rc-1](https://github.com/gradle/gradle/releases/tag/v4.8.0-RC1), as usual I check RCes to report issues earlier

Looks like Bugsnag has some compatibility issue (likely easy to fix, but I don't want to go into details)

```java
Caused by: org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'org.gradle.api.internal.collections.SortedSetElementSource@6db7e212' with class 'org.gradle.api.internal.collections.SortedSetElementSource' to class 'java.util.TreeSet'
	at com.bugsnag.android.gradle.BugsnagPlugin.isJackEnabled(BugsnagPlugin.groovy:262)
	at com.bugsnag.android.gradle.BugsnagPlugin.setupMappingFileUpload(BugsnagPlugin.groovy:106)
	at com.bugsnag.android.gradle.BugsnagPlugin$_applyBugsnagToVariant_closure5.doCall(BugsnagPlugin.groovy:94)
	at com.bugsnag.android.gradle.BugsnagPlugin.applyBugsnagToVariant(BugsnagPlugin.groovy:87)
	at org.gradle.internal.metaobject.BeanDynamicObject$GroovyObjectAdapter.invokeOpaqueMethod(BeanDynamicObject.java:579)
	at org.gradle.internal.metaobject.BeanDynamicObject$MetaClassAdapter.invokeMethod(BeanDynamicObject.java:506)
	at org.gradle.internal.metaobject.BeanDynamicObject.tryInvokeMethod(BeanDynamicObject.java:191)
	at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:78)
	at com.bugsnag.android.gradle.BugsnagPlugin$_apply_closure1$_closure11.doCall(BugsnagPlugin.groovy:42)
	at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:71)
	at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:155)
	at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:106)
	at org.gradle.util.ConfigureUtil$WrappedConfigureAction.execute(ConfigureUtil.java:167)
	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:154)
	at org.gradle.api.internal.DefaultDomainObjectCollection.all(DefaultDomainObjectCollection.java:175)
	at org.gradle.api.DomainObjectCollection$all$0.call(Unknown Source)
	at com.bugsnag.android.gradle.BugsnagPlugin$_apply_closure1.doCall(BugsnagPlugin.groovy:41)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:40)
	at org.gradle.listener.ClosureBackedMethodInvocationDispatch.dispatch(ClosureBackedMethodInvocationDispatch.java:25)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:42)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:230)
	at org.gradle.internal.event.BroadcastDispatch$SingletonDispatch.dispatch(BroadcastDispatch.java:149)
	at org.gradle.internal.event.AbstractBroadcastDispatch.dispatch(AbstractBroadcastDispatch.java:58)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:324)
	at org.gradle.internal.event.BroadcastDispatch$CompositeDispatch.dispatch(BroadcastDispatch.java:234)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:140)
	at org.gradle.internal.event.ListenerBroadcast.dispatch(ListenerBroadcast.java:37)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy30.afterEvaluate(Unknown Source)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$1.execute(LifecycleProjectEvaluator.java:83)
	at org.gradle.configuration.project.LifecycleProjectEvaluator$1.execute(LifecycleProjectEvaluator.java:80)
```

If you think it's an compatibility issue on Gradle side, then this bug should be opened in Gradle repo :)

**Comment by kattrali** [comment]
Fixed by #110

**Comment by artem-zinnatullin** [comment]
That was quick, great work!

**Comment by artem-zinnatullin** [comment]
Hey guys, can you please publish an update with fix? Gradle 4.8 was released :)

**Comment by fractalwrench** [comment]
@artem-zinnatullin this has been released in [v3.2.7](https://github.com/bugsnag/bugsnag-android-gradle-plugin/releases/tag/v3.2.7) :)
